call_apply_bind_exercise/readme.md

```diff
@@ -7,15 +7,26 @@ var obj = {
     fullName: "Harry Potter",
     person: {
         sayHi: function(){
-            return "This person's name is " + this.fullName
+            return "This person's name is " + this.fullName"
         }
     }
 }
 ```
 
+```
+var obj = {
+    fullName: "Harry Potter",
+    person: {
+        sayHi: function(){
+            return "This person's name is " + this.fullName
+        }.bind(obj)
```

Comment:
this won't work because obj isn't defined yet.  You could do:

```
var obj = {
  fullName: "Harry Potter",
  person: {
     sayHi() {
        return `This person's name is ${this.fullName}`;
     }
  }
}

obj.person.sayHi = obj.person.sayHi.bind(obj);
```